dart-parser/src/parser/comment.rs

```diff
@@ -0,0 +1,76 @@
+use std::str;
+
+use nom::{
+    branch::alt,
+    bytes::complete::{is_not, tag},
+    combinator::{cut, eof, recognize},
+    multi::fold_many0,
+    sequence::{preceded, terminated, tuple},
+    IResult, Parser,
+};
+
+use crate::{dart::comment::Comment, parser::common::*};
+
+pub fn comments(s: &str) -> IResult<&str, Comment> {
```

Comment:
I'd prefer singular `comment` here, because it parses a single comment (one single-line or one multi-line, but only one any supported type).